Operational-Procedures/Main-Telescope/Refrigeration-System-Pathfinder/_static/README

```diff
@@ -0,0 +1,23 @@
+This README includes information on using static objects within this directory.
```

Comment:
You may want to remove this README file here

Reply:
Done